homeassistant/components/auth/__init__.py

```diff
@@ -133,7 +134,7 @@
 from homeassistant.components.http.ban import log_invalid_auth
 from homeassistant.components.http.data_validator import RequestDataValidator
 from homeassistant.components.http.view import HomeAssistantView
-from homeassistant.const import HTTP_BAD_REQUEST, HTTP_FORBIDDEN, HTTP_OK
+from homeassistant.const import HTTP_OK
```

Comment:
Why not replace this with `HTTPStatus.OK` ?

Reply:
The whole changeset is quite large, I'm chopping it up into smaller logical pieces for easier review. The topic of this PR is to adjust `HomeAssistantView` function signatures and usages, and the mentioned `HTTP_OK` usage here is not in one of those, it's in aiohttp `web.Response`. It will be changed in a forthcoming PR.